website/settings.py

```diff
@@ -215,7 +215,7 @@
   #'social_auth.backends.browserid.BrowserIDBackend',
   #'social_auth.backends.contrib.live.LiveBackend',
   # Django default this is always needed and must always be last.
-  'django.contrib.auth.backends.ModelBackend',	    
+  'django.contrib.auth.backends.ModelBackend',      
```

Comment:
Addressed in 16955796fc708c38cc5b5fd5a8dc402a389405c3, but there's much more to do.